signbank/settings/base.py

```diff
@@ -214,3 +231,11 @@
 
 mimetypes.add_type("video/mp4", ".mov", True)
 mimetypes.add_type("video/webm", ".webm", True)
+
+# Allow DEBUG to be set using an environment variable
+DEBUG = os.environ.get('DEBUG', 'false').lower() == 'true'
+
+# Set ALLOWED_HOSTS from an environment variable, with defaults
+DEFAULT_ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
+ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS").split(
+    ",") if os.getenv("ALLOWED_HOSTS") else DEFAULT_ALLOWED_HOSTS
```

Comment:
I feel like it would be nicer to provide the default allowed hosts as a default second param rather than with an `if else` here, e.g.

```
ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "127.0.0.1,localhost").split(",")
# or

DEFAULT_ALLOWED_HOSTS = ['127.0.0.1', 'localhost'].join(',')
ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", DEFAULT_ALLOWED_HOSTS).split(",")
```

Reply:
Oh right I see you've just copied this as-written from `signbank/settings/development.py` - feel free to ignore if you want to keep it as-is 😄